pkg/api/types.go

```diff
@@ -926,6 +914,12 @@ const (
 	// Status code 422
 	StatusReasonInvalid StatusReason = "Invalid"
 
+	// StatusReasonTimeout means that the request could not be completed within the given time.
```

Comment:
I'm fine if this is addressed in a separate PR.

@nikhiljindal Please file an issue if you don't fix it in this PR.